viadot/sources/cloud_for_customers.py

```diff
@@ -0,0 +1,257 @@
+import re
+import requests
+import pandas as pd
+
+from copy import deepcopy
+from typing import Any, Dict, List, Optional
+from pydantic import BaseModel
+from urllib.parse import urljoin
+from viadot.exceptions import CredentialError
+
+from ..utils import handle_api_response
+from ..config import get_source_credentials
+from .base import Source
+
+
+class CloudForCustomersCredentials(BaseModel):
+    username: str  # eg. username@{tenant_name}.com
+    password: str
+
+
+class CloudForCustomers(Source):
+    """Cloud for Customers connector to fetch Odata source.
+
+    Args:
+        url (str, optional): The URL to the C4C API. E.g 'https://my396876543.jrt.ondemand.com/c4c/v1/'.
```

Comment:
remove private instance

Reply:
It is not private instance, it is a random example I typed that looks like an actual instance url.
Letters and numbers were chosen randomly, while preserving similar url structure to a real one.

Reply:
.

Reply:
Fixed